Contents/Artifacts/OrbOfAlchemistArtifact.cs

```diff
@@ -25,7 +33,15 @@ class AlchemistKnowledgePlayer : ModPlayer {
 			if (Alchemist) {
 				int lengthNPC = target.buffType.Where(i => i != 0).Count();
 				int lengthPlayer = Player.buffType.Where(i => i != 0).Count();
-				modifiers.FinalDamage *= lengthNPC + Player.buffType.Where(i => i != 0).Count();
+				int finalLength = lengthNPC + Player.buffType.Where(i => i != 0).Count();
+				if (finalLength <= 0) {
```

Comment:
I'm 100% sure `finalLength` will _never_ be lesser than `0`, so you can simplify this to just equality check.

Reply:
alright